### ISSUE TITLE
hello, the question about the accuracy on the test data by using default configs

### ISSUE THREAD

**Issue by fishishappyandfree** [Sleepychord/ImprovedGAN-pytorch#5]
when I run the command, python ImprovedGAN.py --cuda, and set the 100 epochs, the accuracy on test data is only around 65%. that is not same with your mentened 98.5%.

**Comment by Sleepychord** [comment]
Hi, please provide more details. 
Moreover, this repo is constructed two years ago(python 2.7 and pytorch 0.3).  Many things changes in these years. I will soon update to pytorch 1.0+ and python 3.7+.

**Comment by fishishappyandfree** [comment]
after only one epochs, the result is shown as follows:
Iteration 0, loss_supervised = 0.4090, loss_unsupervised = 0.1177, loss_gen = 26.9792 train acc = 0.9804
Eval: correct 6592 / 10000

**Comment by fishishappyandfree** [comment]
my using virtual environment include torch 1.0.1, python 3.6.9

**Comment by fishishappyandfree** [comment]
thank you for your share and answer

**Comment by fishishappyandfree** [comment]
after eleven epochs, the result is shown as follows:
Iteration 10, loss_supervised = 0.0000, loss_unsupervised = 0.0000, loss_gen = 74.4981 train acc = 1.0000
Eval: correct 6144 / 10000
then there are no change after eleventh epoch

**Comment by fishishappyandfree** [comment]
after eleven epochs, the result is shown as follows:
Iteration 10, loss_supervised = 0.0000, loss_unsupervised = 0.0000, loss_gen = 74.4981 train acc = 1.0000
Eval: correct 6144 / 10000
then there are no change after eleventh epoch except the increasing loss_gen.

**Comment by Sleepychord** [comment]
Hi, I have updated the codes for pytorch 1.0. Please try again.

**Comment by fishishappyandfree** [comment]
thank you for your help.

**Comment by daixiangzi** [comment]
> Hi, I have updated the codes for pytorch 1.0. Please try again.
i have other problem:,about unlabel_loss compute,
in paper :
![图片](https://user-images.githubusercontent.com/24811131/69725125-5fc8a280-1158-11ea-891f-47f5a9c28403.png)
code:
-torch.mean(logz_unlabel) + torch.mean(F.softplus(logz_unlabel))
F.softplus(logz_unlabel) ==log(1+exp(....))??

**Comment by Sleepychord** [comment]
Hi, @daixiangzi . What's the problem? Of course `softplus(x)=log(1+exp(x))`. You should ask Google or some other documents for the definitions, if this is your question.

**Comment by daixiangzi** [comment]
very sorry，i am looking in the wrong. again thank you。。



---Original---

**Comment by daixiangzi** [comment]
> thank you for your help.
hello,do you try to run this code on cifar10?

**Comment by daixiangzi** [comment]
> Hi, @daixiangzi . What's the problem? Of course `softplus(x)=log(1+exp(x))`. You should ask Google or some other documents for the definitions, if this is your question.

hello,do you try to run this code on cifar10?

**Comment by Sleepychord** [comment]
No, I didn't.
